### ISSUE TITLE
p-columnFilter: missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#14608]
The main ToggleButton still missing aria-label. [There is wrong translation keys set in api.](https://github.com/primefaces/primeng/commit/12ca541981cf7c1b2d0c09b85fea1eb4b66608ed#r137434755)

_Originally posted by @TaneliTuomola in https://github.com/primefaces/primeng/issues/14537#issuecomment-1900447454_

Tested with primeNg 17.3.3: https://primeng.org/table#filter
